test/components/view-toggle/view-toggle.vdiff.js

```diff
@@ -0,0 +1,41 @@
+import '../../../src/components/view-toggle/view-toggle.js';
+import { expect, fixture, html } from '@brightspace-ui/testing';
+
+const basicFixture = html`<d2l-labs-view-toggle
+toggle-options='[{"text":"Bananas","val":"overall"},{"text":"Minions","val":"minios"},{"text":"Pyjamas","val":"subject"}]'
+text="Toggle: "></d2l-labs-view-toggle>`;
+
+const noWrapFixture = html`<d2l-labs-view-toggle
+toggle-options='[{"text":"Bananas","val":"overall"},{"text":"Minions","val":"minios"},{"text":"Pyjamas","val":"subject"},{"text":"Elephant","val":"longmemory"},{"text":"Tiger","val":"fast"}]'
+text="Toggle: "></d2l-labs-view-toggle>`;
+
+const viewports = [
+	{ viewport: { height: 800, width: 1230 } },
+	{ viewport: { height: 800, width: 865 } },
+	{ viewport: { height: 800, width: 767 } },
+	{ viewport: { height: 800, width: 614 } },
+	{ viewport: { height: 800, width: 430 } }
+];
+
+describe('d2l-labs-view-toggle', () => {
+	it('should not wrap', async() => {
```

Comment:
The snapshot of this test is the same as the mobile viewport one, but this does lock in a test specifically for a too-long set of toggles. Maybe I should update the basicFixture to just be two toggles?